addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -85,8 +85,15 @@ QUnit.test('composer text input: basic rendering', async function (assert) {
     );
     assert.strictEqual(
         document.querySelector(`.o_ComposerTextInput_textarea`).placeholder,
-        "Write something...",
-        "should have placeholder in note editable of composer text input"
+        "Send a message to followers...",
+        "should have 'Send a message to followers...' as placeholder composer text input"
+    );
+
+    await afterNextRender(() => composer.update({ isLog: true }));
```

Comment:
This is a component test, so don't use public method on records

Reply:
No possibility in the interface (without having a chatter) to switch from comment to note... Do you prefer I make two different tests ?

Reply:
I would make a chatter test then.

Also this shows chatter topbar contains info that are composer-specific... In fact, almost whole chatter model contain thread-specific (so contain composer-specific), which is a motivation to move most `mail.chatter` fields to `mail.thread` (task-2243518)

Reply:
I made another test as the switching is not important to test here, mostly the placeholder. there is already a test in chatter to test the switch